src/components/Staff/ManagePromotion.vue

```diff
@@ -0,0 +1,347 @@
+<template>
+  <v-container>
+    <v-data-table :headers="promoHeaders" :items="promoList" class="elevation-1">
+      <template v-slot:top>
+        <v-toolbar color="black orange--text text--darken-2" flat>
+          <v-toolbar-title color="orange darken-2">Promotions</v-toolbar-title>
+          <v-spacer></v-spacer>
+          <v-col cols="4" md="3">
+            <v-row justify="end">
+              <v-btn color="orange darken-2" dark class="mb-2" @click="getPromos">Refresh</v-btn>
+            </v-row>
+          </v-col>
+
+          <v-dialog v-model="dialog" max-width="700px">
+            <template v-slot:activator="{ on }">
+              <v-col cols="4" md="2">
+                <v-row justify="end">
+                  <v-btn
+                    color="orange darken-2"
+                    dark
+                    class="mb-2"
+                    @click="resetDates"
+                    v-on="on"
+                  >New Item</v-btn>
+                </v-row>
+              </v-col>
+            </template>
+            <v-card>
+              <v-card-title>
+                <span class="headline">{{ formTitle }}</span>
+              </v-card-title>
+
+              <v-form ref="form" v-model="validForm" lazy-validation>
+                <v-card-text>
+                  <v-container>
+                    <v-row>
+                      <v-col cols="12" sm="6" md="4">
+                        <v-menu
+                          ref="menu"
+                          v-model="menu"
+                          :close-on-content-click="false"
+                          transition="scale-transition"
+                          offset-y
+                          max-width="290px"
+                          min-width="290px"
+                        >
+                          <template v-slot:activator="{ on }">
+                            <v-text-field
+                              required
+                              :rules="rules.dates"
+                              v-model="computedDateFormatted"
+                              label="Choose start and end date (read only text field)"
```

Comment:
"(read only text field)"
Is this supposed to be here?

Reply:
yeah, the user cannot manually enter the date. should i remove it?